.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "build"]
 	path = build
-	url = https://github.com/upbound/build
+	url = https://github.com/soorena776/build
```

Comment:
the `soorena776` change is temporary and will be reverted once https://github.com/upbound/build/pull/74 is completed